### DISCUSSION TITLE
Should I follow release branches or main branch to get the latest release?

### DISCUSSION THREAD

**Discussion by alperkar** [actions/virtual-environments#7089] [General questions]
I have a fork of this repository in my Azure DevOps organization and want to keep my repo up to date with the latest releases. Should I use the main branch or release branches? In other words, does the main branch contain unreleased changes? If yes I wouldnt really want to build images from main branch.

**Comment by mikhailkoliada** [comment]
Releases are the weekly state of the main branch, given that we are installing lots of latest software, main is always up to date.
